extensions/interactions/GraphInput/directives/graph_viz_directive.html

```diff
@@ -41,7 +42,8 @@
     </svg>
 
     <g ng-repeat="button in buttons">
-      <rect width="140"
+      <rect class = "protractor-test-<[(button.description | translate).replace(' ','-')]>-button"
```

Comment:
Remove spaces before and after "="



(@apb7 @kevinlee12  we should catch this in linter.)

Reply:
Done!